packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1573,32 +1579,44 @@ class StackViewport extends Viewport implements IStackViewport {
 
     // 1. Update the pixel data in the vtkImageData object with the pixelData
     //    from the loaded Cornerstone image
-    const pixelData = image.getPixelData();
```

Comment:
Please undo this line - it isn't needed as the assignment to pixelData is never used.